frontend/apps/os-shell/src/styles/terrafusion-quantum-animations.css

```
@@ -220,18 +220,18 @@ Reliability Through Quantum-Inspired Motion Design
 @keyframes neural-pathway-activation {
   0% {
     stroke-dashoffset: 1000;
-    stroke: hsl(207 100% 50% / 0.2);
+    stroke: hsl(var(--tf-network-blue-hs) var(--tf-l-50) / 0.2);
     filter: blur(2px);
   }
 
   50% {
-    stroke: hsl(180 100% 50% / 0.8);
+    stroke: hsl(var(--tf-transcend-cyan-hs) var(--tf-l-50) / 0.8);
     filter: blur(0px) glow(5px);
```

Biome (2.4.4)

[error] Unexpected unknown function: glow

(lint/correctness/noUnknownFunction)